torch/csrc/jit/script/compiler.cpp

```diff
@@ -1984,6 +2219,13 @@ struct to_ir {
         return emitShortCircuitIf(
             tree->range(), inputs[0], inputs[1], tree->kind() == TK_OR);
       }
+      case TK_IS:
```

Comment:
This feels like a lot of duplication just to add the setIsBoolInfo flag at the end. Refactor?